tensorflow/c/c_api.cc

```diff
@@ -782,7 +782,7 @@ void TF_GraphGetTensorShape(TF_Graph* graph, TF_Output output, int64_t* dims,
 
 extern "C" {
 
-static TF_OperationDescription* TF_NewOperationLocked(TF_Graph* graph,
+TF_OperationDescription* TF_NewOperationLocked(TF_Graph* graph,
                                                       const char* op_type,
```

Comment:
Could you please indent this and others. The linter might complain during merge.

Reply:
Do you mean I should indent everything in the `extern "C" {` block?  Or just the functions I changed (the other's aren't)?  I went through and lined up all the parameters.

Reply:
Yep, I meant the params. Thanks for fixing.